pagopa/io-app
# chore: [IOBP-716] Empty state payment home alignment

## Short description
This PR fixes alignment in case of empty state within the payment home screen

## List of changes proposed in this pull request
- Added a flex to the main View

## How to test
- Enable the new wallet landing page FF
- If you don't have any transactions and any payment method, you should see the correct alignment of the text


## Preview
|Before|After|
|-|-|
|<img src="https://github.com/pagopa/io-app/assets/34343582/f64a5e7f-85c6-47be-8fe0-6811f9eb8aa3" width="300" />|<img src="https://github.com/pagopa/io-app/assets/34343582/fe0ef47c-76d7-4e15-84da-68491b9665ae" width="300" />|

|Before|After|
|-|-|
|<img src="https://github.com/pagopa/io-app/assets/34343582/5b9efc11-af7e-4008-8a05-af9613e3900b" width="300" />|<img src="https://github.com/pagopa/io-app/assets/34343582/e5075062-19ef-43d1-9cc2-05e05cfdf73b" width="300" />|
